tests/nnstreamer_sink/unittest_sink.cpp

```diff
@@ -1100,6 +1100,8 @@ TEST (tensor_sink_test, signals)
       (GCallback) _eos_cb, NULL);
   EXPECT_TRUE (handle_id > 0);
 
+  g_object_set (g_test_data.sink, "sync", FALSE, NULL);
```

Comment:
If this can fix the time issue, I think it is better to add this in _setup_pipeline()

Reply:
I added a common test option, `need_sync`.